commons-mongo/src/main/scala/com/avsystem/commons/mongo/async/MongoObservableReactivePublisher.scala

```diff
@@ -0,0 +1,42 @@
+package com.avsystem.commons
+package mongo.async
+
+import com.mongodb.async.{client => mongo}
+import monix.execution.atomic.AtomicBoolean
+import org.{reactivestreams => reactive}
+
+final class MongoObservableReactivePublisher[T](observable: mongo.Observable[T]) extends reactive.Publisher[T] {
+  def subscribe(subscriber: reactive.Subscriber[_ >: T]): Unit = {
+    observable.subscribe(
+      new mongo.Observer[T]() {
+        override def onSubscribe(subscription: mongo.Subscription): Unit = {
+          subscriber.onSubscribe(new reactive.Subscription() {
+            private final val cancelled: AtomicBoolean = AtomicBoolean(false)
+
+            def request(n: Long): Unit = {
+              if (!subscription.isUnsubscribed && n < 1) {
```

Comment:
Maybe make the condition literally like the documentation says, `<= 0` for more hmm... consistency.